Common/include/toolboxes/multilayer_perceptron/CLayer.hpp

```diff
@@ -0,0 +1,146 @@
+/*!
+ * \file CLayer.hpp
+ * \brief Declaration of artificial neural network interpolation class
+ * \author E. Bunschoten
+ * \version 7.5.0 "Blackbird"
```

Comment:
In other files too please.
```suggestion
 * \version 7.5.1 "Blackbird"
```